crates/cadenza-eval/src/eval.rs

```diff
@@ -107,14 +100,9 @@ fn eval_literal(lit: &Literal) -> Result<Value> {
 }
 
 /// Evaluates an identifier by looking it up in the environment.
-fn eval_ident(
-    ident: &Ident,
-    interner: &mut Interner,
-    env: &Env,
-    compiler: &Compiler,
-) -> Result<Value> {
-    let text = ident.syntax().text().to_string();
-    let id = interner.intern(&text);
+fn eval_ident(ident: &Ident, env: &Env, compiler: &Compiler) -> Result<Value> {
+    let text = ident.syntax().text();
+    let id: InternedString = text.to_string().as_str().into();
```

Comment:
I guess Rowan doesn't allow getting at the `&str` then? Let's at least put a Todo and update the status doc to investigate how to work around that issue.